yarn.lock

```diff
@@ -512,7 +512,78 @@
     protobufjs "^6.11.2"
     uint8arrays "^3.0.0"
 
-"@chainsafe/persistent-merkle-tree@^0.4.2":
+"@chainsafe/lodestar-api@^0.37.0":
```

Comment:
@wemeetagain do you know why now local packages show up in the yarn.lock? This happens if I run `yarn` locally too